moveit_cartesian_plan_plugin/src/add_way_point.cpp

```diff
@@ -5,19 +5,19 @@
 namespace moveit_cartesian_plan_plugin
 {
 
-AddWayPoint::AddWayPoint(QWidget *parent):rviz::Panel(parent), tf_()
+AddWayPoint::AddWayPoint(QWidget *parent):rviz::Panel(parent)//, tf_()
 {
-    /*!  The constructor sets the Object name, resets the Interactive Marker server. 
+    /*!  The constructor sets the Object name, resets the Interactive Marker server.
          It initialize the subscriber to the mouse click topic and registers the call back to a mouse click event.
          It initializes the constants for the Marker color and scales, all the Interactive Markers are defined as arrows.
-         The User Interaction arrow is set to have red color, while the way-points when in an IK Solution are set as blue. 
+         The User Interaction arrow is set to have red color, while the way-points when in an IK Solution are set as blue.
          The Way-Points which are outside an IK solution are set to have yellow color.
     */
      setObjectName("CartesianPathPlannerPlugin");
      server.reset( new interactive_markers::InteractiveMarkerServer("moveit_cartesian_plan_plugin","",false));
-     point_sub_.subscribe(n_, "/clicked_point", 10);
-     tf_filter_ = new tf::MessageFilter<geometry_msgs::PointStamped>(point_sub_, tf_, target_frame_, 10);
-     tf_filter_->registerCallback( boost::bind(&AddWayPoint::msgCallback, this, _1) );
+    //  point_sub_.subscribe(n_, "/clicked_point", 10);
```

Comment:
Commented out code.  Can this be removed?  Out of curiosity, why is `tf_fileter` no longer needed.